### PR TITLE
Run content style checker on integration tests

### PR DESCRIPTION
**Pull request by warp-core**
**Feature:**

## Feature Details
Runs the content style check script on the integration test data files and fixes a few issues it found and adds copyright headers to all the integration test files (except the one that just disables missions because it's just a list of missions.)

## Testing Done
It tests itself?

### Automated Tests Added
It is an automated test.

### CODE REVIEW

File: tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# Copyright (c) 2022 by Hurleveur, an anonymous author
```
Comment (Hurleveur) [inline tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt:1]: Why did you add anonymous author here but not for the rest? Not that it matters much but we should prob have conventions (or does it mean me and an anonymous author?)

Comment (warp-core) [reply tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt:1]: You and someone else.
I think @UmorderedSigh was involved in writing this file? This is how she refers to herself in copyright headers. Assuming she wants to be listed here.

Comment (Hurleveur) [reply tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt:1]: I think it would be clearer with an and in that case
Also we dont really have to mention amonymous authors?

This doesnt matter much either way, it just feels weird

Comment (warp-core) [inline tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt:1]:
```suggestion
# Copyright (c) 2022 by Hurleveur and an anonymous author
```

Comment (UnorderedSigh) [reply tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt]: I'm still an author, whether I'm anonymous or not.

**Review by UnorderedSigh — APPROVED** [review]
Good addition. The style checker sometimes finds genuine coding errors. It'll be nice to have it in the integration tests.

File: tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt (hunk 1 above)
Comment (Hurleveur) [reply tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt]: Yeah so are everyone adding content to all files, we just dont mention them anyway, and I never noticed by an anonymous author, declining copyright often goes to MZ or whoever else wrote the file?
This doesnt matter much my issue with it was mosty the confusion that it said `Hurleveur, an anonymous author`

**Review by Hurleveur — APPROVED** [review]

File: tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt (hunk 1 above)
Comment (UnorderedSigh) [inline tests/integration/config/plugins/integration-tests/data/tests/tests_conditional_choice.txt]:
```suggestion
# Copyright (c) 2022 by Hurleveur and an anonymous author
```
